lib/ts/recipe/session/index.ts

```diff
@@ -44,6 +44,7 @@ export default class SessionWrapper {
         userId: string,
         accessTokenPayload: any = {},
         sessionData: any = {},
+        useDynamicAccessTokenSigningKey?: boolean,
         userContext: any = {}
```

Comment:
We should remove updateAccessTokenPayload since this is a breaking change anyway, and we do not want people to call updateAccessTokenPayload thinking that it's a merge.